### DISCUSSION TITLE
best practice auth for nextjs

### DISCUSSION THREAD

**Discussion by Manubi** [vercel/next.js#20859] [Help] [answered]
Hi,
what do you consider best practice when using auth with nextjs? I am currently using firebase auth but I am not that happy with it and think about changing to next-auth? Mainly because firebase is pretty slow (I am base in europe) and I think next-auth is easier to use with SSR.

How do you handle auth and are you happy with your choice?

**Comment by jamesmosier** [comment, marked as answer]
Good timing! This was just posted today: https://nextjs.org/docs/authentication
